beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events.md

```diff
@@ -0,0 +1,154 @@
+# Rules for XDM events in Adobe Experience Launch
+
+{% hint style="warning" %}
+The Adobe Experience Platform Edge mobile extension is currently in BETA. Use of this extension is by invitation only. Please contact your Adobe Customer Success Manager to learn more and get access to the materials for this tutorial.
+{% endhint %}
+
+## Prerequisites for this tutorial
+
+* Access to Adobe Experience Platform
+* Minimal Swift / Android development knowledge 
+* Knowledge about the AEP Edge extension
+* Completion of [Assignment 1 - AEP Edge extension setup and XDM usage](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-1-edge-extension-setup) (required to have a mobile property in Launch with Edge configuration setup).
+* Completion of [Assignment 3 - XDM implementation](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events) (required to have a Product reviews schema and dataset).
+
+### Download the sample application
+
+**Note:** This step is not required if you already implemented the steps described in [Assignment 3 - XDM implementation](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events).
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Java
+
+Download the Android Sample application from [https://github.com/adobe/aepsdk-sample-app-android/tree/beta-assignment-4](https://github.com/adobe/aepsdk-sample-app-android/archive/beta-assignment-4.zip).
+
+To get started, follow the steps described in [AEP SDK Sample App Android - Installation](https://github.com/adobe/aepsdk-sample-app-android/tree/beta-assignment-4#installation).
+{% endtab %}
+
+{% tab title="iOS" %}
+
+#### Swift
+
+Download the iOS Swift Sample application from [https://github.com/adobe/aepsdk-sample-app-ios/tree/beta-assignment-4](https://github.com/adobe/aepsdk-sample-app-ios/archive/beta-assignment-4.zip).
+
+To get started, follow the steps described in [AEP SDK Sample App Swift - Installation](https://github.com/adobe/aepsdk-sample-app-ios/tree/beta-assignment-4#installation).
+{% endtab %}
+{% endtabs %}
+
+### Set up the required fields
+
+**Note:** The steps in this section are not required if you already implemented the steps described in [Assignment 3 - XDM implementation](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-4-rules-xdm-events).
```

Comment:
URL link for `Assignment 3 - XDM implementation` should be https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation